ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml

```diff
@@ -246,6 +246,24 @@ presubmits:
         resources: {}
       serviceAccountName: ci-operator
     trigger: (?m)^/test( | .* )upgrade-v7.0,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```

Comment:
If we add this in this PR, we should set this to `false` for now until we actually tag the `v8.0` release, otherwise this is going to fail and block merges.

Reply:
I have just removed it in the new commit

Reply:
New commit looks the same as the old commit to me, unless I'm missing something...